deltachat-ffi/src/lib.rs

```diff
@@ -703,6 +706,7 @@ pub unsafe extern "C" fn dc_event_get_data2_str(event: *mut dc_event_t) -> *mut
         | EventType::SecurejoinJoinerProgress { .. }
         | EventType::ConnectivityChanged
         | EventType::SelfavatarChanged
+        | EventType::ConfigSynced { .. }
```

Comment:
shouldn't that return the key name? even though it is currently not used in cffi, if we do the effort to add things to cffi, it should be consistent.